src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+
+  let sum = days * costPerDay;
+
+  if (days >= 7) {
+    sum -= discountSevenDays;
+  } else if (days >= 3) {
+    sum -= discountThreeDays;
+  }
```

Comment:
You should return value right after you've found the right condition. It makes the code simpler to read and helps to avoid unnecessary `else if`. And using this approach you can use a `sum` as a `const`, cause there's no need to change it anymore.
```suggestion
  if (days >= 7) {
    return sum - discountSevenDays;
  }
  
  if (days >= 3) {
    return sum - discountThreeDays;
  }
```